src/main/java/io/sentry/UploadSourceBundleMojo.java

```diff
@@ -71,7 +71,8 @@ public class UploadSourceBundleMojo extends AbstractMojo {
 
   @Override
   public void execute() throws MojoExecutionException {
-    if (skip || skipSourceBundle) {
+    final @Nullable String sentrySkipSourceUpload = System.getenv("SENTRY_SKIP_SOURCE_UPLOAD");
```

Comment:
Belongs to PR https://github.com/getsentry/sentry-maven-plugin/pull/65/files.
Maybe revert this part, as it will be added by the aforementioned PR

Reply:
Yeah I wanted to merge the other one first, then merge main in here and merge this after.